app/models/email.rb

```diff
@@ -1,3 +1,11 @@
+# == Schema Information
```

Comment:
Well for me "Schema" refers to the database schema, but its okay for me when we dont change it